### PR TITLE
[About Us] Complete About Us Page

### PR DESCRIPTION
**Pull request by IcedTea2K**
Features:
- Assemble the "About Us" page. Should be accessible a `/about-us`
- Add infinite scrolling carousel. Planning on adding videos/gif for the photos, but would have to consult this with captains. The carousel slows down whilst hovered.
- Add "Hear our stories" section for some of biomod's story
- Also add call to action, asking people to join us.

Some references:
image
image
image

### CODE REVIEW

File: src/app/about-us/page.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"use client";
+import CallToAction from "@/components/about-us/CallToAction";
+import Hero from "@/components/about-us/Hero";
+import Stories from "@/components/about-us/Stories";
+import Team from "@/components/about-us/Team";
+
+export default function AboutUs() {
+  return (
+    <div className="flex flex-col items-center gap-y-defaultMob lg:gap-y-default bg-bg2 bg-no-repeat">
```
Comment (daniel-panhead) [inline src/app/about-us/page.tsx]: It looks like there's duplicate backgrounds? If you take out `bg-bg2` there's still a background

File: src/components/about-us/Team.tsx
Comment (daniel-panhead) [inline src/components/about-us/Team.tsx:1]: soooo sexy

**Review by daniel-panhead — APPROVED** [review]
goated, lgtm except the weird duplicate background thing

File: src/app/about-us/page.tsx (hunk 1 above)
Comment (IcedTea2K) [reply src/app/about-us/page.tsx]: oh, i remember copying this from other pages.....

**Comment by CarsonKiibi** [comment]
lgtm, q tho - for cards, what will content underneath name be if team is already in the little pills at the bottom? 

The some of our stories thing is really good idea it looks great and I think people will actually read it if they wanna join or somethin

**Pull request merged**